modules/internal/Atomics.chpl

```diff
@@ -17,36 +17,56 @@
  * limitations under the License.
  */
 
-// See runtime/include/atomics/README for more info about these functions
```

Comment:
I think these are still useful developer comments, can they just be converted to `//` comments with a comment above a saying `Developer Notes:`?